tests/steps/application_v2_steps.py

```diff
@@ -185,6 +185,23 @@ def search_application2(context, creator):
     context.response = context.icl.search_applications(creator=creator)
 
 
+@when(
+    'we make a LookupApplicationBoxByIDandName call with applicationID {app_id} with encoded box name "{box_name}"'
+)
+def lookup_application_box(context, app_id, box_name):
+    boxes = split_and_process_app_args(box_name)[0]
```

Comment:
in both py/js-sdk, this `split_and_process_app_args` is applied to parse `boxName`, while in go/java-sdk, this is not the case, for `name` does simply put string into params in query.

Do we want to unify the sdk behaviors in this case?

Reply:
This is deliberate because the Java/Go SDKs have different inputs due to generated code. The JS/Py SDKs' APIs take box name as bytes instead of strings.